project/models.py

```diff
@@ -0,0 +1,69 @@
+import datetime
+
+
+incidents = []
+users = []
+
+
+class Incident:
+    """class for incidents"""
+
+    def __init__(self, created_by, incident_type, location, file, comment):
+        self.incident_id = len(incidents)+1
+        self.date = datetime.date.today()
+        self.created_by = created_by
+        self.incident_type = incident_type
+        self.location = location
+        self.status = "draft"
+        self.file = file
+        self.comment = comment
+
+    def to_json(self):
+        """ 
+        method to turn incident to dictionary
+        """
+        return {
+            "incident_id": self.incident_id,
+            "date": self.date,
+            "created_by": self.created_by,
+            "incident_type": self.incident_type,
+            "location": self.location,
+            "status": self.status,
+            "file": self.file,
+            "comment": self.comment
+        }
+
+
+class User:
+    """
+    class for users
+    """
+    def __init__(self, *args):
```

Comment:
use keyword arguments here
```suggestion
    def __init__(self, **kwargs):
```